modules/serverless-metering-queries.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+// serverless-metering.adoc
+
+[id="queries-metering-serverless_{context}"]
+= Queries for Knative Serving metering
+The following `ReportQuery` resources are available for use with metering and the {ServerlessProductName} Knative Serving component.
```

Comment:
We might want to signify here that these reference the example DataSources from above. In fact I'm wondering if we can fold both ReportQueries into just one object querying both CPU and RAM. @rhuss maybe knows. The two ReportQueries are almost identical so that might seem confusing.

Reply:
I don't think its possible as the 'value' column holding the report metrics (cpu or storage) has different types (`bytes_seconds` for memory, `cpu_core_seconds` for CPU) and I don't know if this can be parameterized as input parameters for the report itself.

And even if it could, it puts then more burden on the user to provide these parameters in the actual `Report` instance which calls this `ReportQuery`. So in this case, I think the duplication is ok.